src/components/user-menu/style.scss

```diff
@@ -81,3 +81,22 @@ a > smoothly-icon:nth-child(1) {
 		display: block;
 	}
 }
+:host[is-open] > div {
+  animation-duration: 0.2s;
+  animation-name: slidein;
+}
+:host:not([is-open]) > div {
+	display: none;
+}
+
+@keyframes slidein {
+  from {
+		transform: translateX(100%);
+		opacity: 0;
+  }
+
+  to {
+		opacity: 1;
+    right: var(--menu-offset-right);
```

Comment:
This line is redundant since the `right` is already set to `var(--menu-offset-right)` and you're not changing the attribute in the animation.